RedHat-UX/red-hat-design-system
# feat(stat): implement `rh-stat`

## What I did

1. Created a component for `rh-stat` based off of the design specs located at https://xd.adobe.com/view/067ac1e2-5343-4dd7-adf4-ef3e18a91387-5329/. 

## Issue Link
(#281)

## Testing Instructions

1. Deploy preview available at https://deploy-preview-330--red-hat-design-system.netlify.app/components/statistic/
2. There are currently spec.ts tests for the component to make sure the slots are loaded

## Notes to Reviewers
1. There are two sets of demos, one without a theme and the other with the dark theme added through a `pfe-band` around the component.
